operator/docs/kit.cloudformation.yaml

```diff
@@ -49,10 +27,21 @@ Resources:
               - "autoscaling:DeleteAutoScalingGroup"
               - "autoscaling:UpdateAutoScalingGroup"
               - "autoscaling:SetDesiredCapacity"
+              - "iam:CreateRole"
```

Comment:
If you need these to create role, then could you also these additional permissions here to maintain parity- https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/testbed/addons/kit/construct.ts#L33-L51

Reply:
I was thinking of cutting a separate PR, but I can push these changes to this one.